components/inputs/input-time.js

```diff
@@ -122,47 +122,43 @@ class InputTime extends LitElement {
 	render() {
 		getIntervals();
 		const input = html`
-			<d2l-dropdown>
-				<div
-					role="combobox"
-					aria-owns="${this._dropdownId}"
-					class="d2l-dropdown-opener"
-					aria-expanded="false">
-					<input
-						aria-controls="${this._dropdownId}"
-						aria-label="${ifDefined(this._getAriaLabel())}"
-						@change="${this._handleChange}"
-						class="d2l-input"
-						?disabled="${this.disabled}"
-						@keypress="${this._handleKeypress}"
-						.value="${this._formattedValue}">
-				</div>
-				<d2l-dropdown-menu id="dropdown" no-padding-footer min-width="195">
-					<d2l-menu
-						id="${this._dropdownId}"
-						role="listbox"
-						class="d2l-input-time-menu"
-						aria-label="${ifDefined(this.label)}"
-						@d2l-menu-item-change="${this._handleDropdownChange}">
-						${INTERVALS.map(i => html`
-							<d2l-menu-item-radio
-								text="${formatTime(i)}"
-								value="${formatValue(i)}"
-								?selected=${this._value === formatValue(i)}>
-							</d2l-menu-item-radio>
-						`)}
-					</d2l-menu>
-					<div class="d2l-input-time-timezone d2l-body-small" slot="footer">${this._timezone}</div>
-				</d2l-dropdown-menu>
-			</d2l-dropdown>
+			<label>
+				<span class="${this.label && !this.labelHidden ? 'd2l-input-label' : 'd2l-offscreen'}" id="${this._dropdownId}-label">${this.label}</span>
```

Comment:
Maybe we want `label` to be required eh? Like throw a `console.warn` in if it's not provided. Because here it'll render an offscreen empty label if it doesn't exist, which is kind of pointless.